examples/docker/pyproject.toml

```diff
@@ -1,16 +1,44 @@
 [build-system]
-requires = ["hatchling"]
-build-backend = "hatchling.build"
+requires = ["flit_core>=3.2,<4"]
```

Comment:
What's the reasoning behind switching to flit?
Isn't flit old-school nowadays?

Reply:
Yeah, so I would rather use, hatch but currently because of PEP517 mismatch uv is currently not supporting the `prepare_metadata_for_build_wheel`. See: https://github.com/astral-sh/uv/issues/2130. I'd rather not use `setuptools` cause it makes the `.egg-info` in the source and because of legacy reasons. Do you know of another good build backend?